### DISCUSSION TITLE
Bots don't use diving suits in certain situations

### DISCUSSION THREAD

**Discussion by SchwarzFuchs** [FakeFishGames/Barotrauma#14325] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Bot's don't use diving suits when they need diving gear and diving mask aren't presented on the submarine.

### Reproduction steps

1. Don't have any diving masks on submarine
2. Make a hole in you submarine big enough so it will require to use diving gear to fix.
3. Order bot to fix it
4. It will say "Need to find diving suit!" and won't do anything after

Video:
[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/ilCL9iSIdZs/0.jpg)](https://www.youtube.com/watch?v=ilCL9iSIdZs)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Probably same bug as [here](https://github.com/FakeFishGames/Barotrauma/discussions/13845)
